src/Manifest.cc

```diff
@@ -614,8 +614,10 @@ GitDependency::install() const {
 DepMetadata
 SystemDependency::install() const {
   const std::string pkgConfigVer = versionReq.toPkgConfigString(name);
-  const std::string cflagsCmd = "pkg-config --cflags '" + pkgConfigVer + "'";
-  const std::string libsCmd = "pkg-config --libs '" + pkgConfigVer + "'";
+  const Command cflagsCmd =
+      Command("pkg-config").addArg("--cflags").addArg(pkgConfigVer);
```

Comment:
Since we are passing arguments directly by `execvp`, quotation for shell must be stripped.
pkgconfig emits error like `'tbb was not found` otherwise.

Reply:
Ah, that makes sense. Thank you!